src/Catalyst.Core.Modules.Dfs/Controllers/V0/SwamController.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Dfs.CoreApi;
+using Catalyst.Core.Lib.Util;
+using Lib.P2P;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Catalyst.Core.Modules.Dfs.Controllers.V0
+{
+    /// <summary>
+    ///   Addresses for peers.
+    /// </summary>
+    public class AddrsDto
+    {
+        /// <summary>
+        ///   Addresses for peers.
+        /// </summary>
+        public Dictionary<string, List<string>> Addrs = new Dictionary<string, List<string>>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)